contrib/vs2019/README.md

```diff
@@ -84,14 +84,14 @@ heirarchy and compile correctly.
 
```

Comment:
Honestly, though I can see your point in motivation/background/philosophy and I also disliked the removal of the VS solution files from the repository in favour of CMake, I dislike the personal note in the Readme file. I'd rather prefer to only have technical relevant information in this file and no subjective opinions.

Reply:
I can take out the portion where I talk about how and where I am planning to use exiv2 and put that in my (not yet there) repository that deals with that aspect. However, I am not in favour of removing my opinion of why I created these project files. People need to know.

Reply:
@sridharb1 I agree with both you and @tbeu.  There are two repositories in use.  In your GitHub repos you can say whatever you wish.  In the Exiv2 repos, I think you could say less and focus on solid technical information.  Please leave something about the rationale as you may find more enthusiastic supporters to bring the msvc/ build environment back to life.  The License can be removed from the Exiv2 repos and retained in your GitHub repos.

I am sucked down into test suite changes with unexpected issues with the unit tests on some platforms.  While I'm working on that (for a couple of days), it would be very helpful if you could update README.md with the directory layout map and other changes we have discussed (concerning vc14, build and working with all the trees).

I understand that you haven't reproduced the issues I have experienced on a "clean machine".  Perhaps you could take a clean directory on your machine such as `c:\Projects\github\exiv2\0.27-maintenance` and experience `contrib\msvc` as it will be seen by Exiv2 users.   I understand that this is painstaking and time-consuming work.  I understand because that's what I have to do with 100% of the Exiv2 release on every platform.

Reply:
Robin,

Updating the documentation will be quick. The information is already there, it just needs to be presented differently.

Unfortunately, I don't have a "clean" machine. Doing it in a new folder will not help, I am positive. My machine resources prevent me from being able to test in a virtual box etc.

I understand that there have been other issues that have been taking time. I can wait. Let's get through the openssl issue that is blocking movement first.

Reply:
I don't want to work on openssl.  I'm already working 7 days a week on RC1.  I've already spent a day on `contrib/vs2019`.  If it doesn't work, it can't ship.

Reply:
That's fine, Robin. I understand not wanting to work on openssl. Let it not ship. People who still want this solution can find it on the web, albeit with more searching.

Reply:
I'll put it on RC1 and find time after that to investigate it.  If I haven't got this working well on my clean machine by the end of May, I'll regretfully have to pull it.  It would be very helpful for you to update the documentation for RC1.

I can't have anything on the release that I don't 100% understand because there will be a support query from Arnold or somebody.

Reply:
No need to mention regret. I absolutely stand by your decision to not include things that have unexplained issues in another machine that we have not been able to resolve.

Ok, I was waiting to see if there were any other issues that you might find and update the documentation in one shot, but it looks like that's not going to happen. I will update this and check into my repository, as well as this PR.

Reply:
Thanks, Shridhar.  Please update README.md and remove the license.  I will get openssl fixed and your stuff will work in RC2.  I always fix things!

I've run into a whole lot of stuff I didn't expect with Unix.  We never expect the unexpected!  I never learn.  Every time I make a release, I say "Never again".

Reply:
Done.